src/vaadin-tooltip.html

```diff
@@ -35,6 +57,72 @@ <h1>Hello World!</h1>
         static get is() {
           return 'vaadin-tooltip';
         }
+
+        static get properties() {
+          return {
+            /**
+             * The text content of the tooltip
+             */
+            for: {
+              type: String,
+              observer: '_attachToTarget'
+            }
+          };
+        }
+
+        _attachToTarget() {
+          const target = this.target();
+
+          if (target) {
+            target.addEventListener('mouseenter', this.show.bind(this));
+            target.addEventListener('focus', this.show.bind(this));
+            target.addEventListener('mouseleave', this.hide.bind(this));
+            target.addEventListener('blur', this.hide.bind(this));
+            target.addEventListener('tap', this.hide.bind(this));
+          }
+
+          this.addEventListener('mouseenter', this.hide.bind(this));
+        }
+
+        _detachFromTarget() {
```

Comment:
Where do you use this function?

Reply:
Wrote it to call it at some point in the future. Do you suggest I remove it from this branch?

Reply:
No, if you plan to use it, then it is ok